app.json

```diff
@@ -28,6 +28,14 @@
       "description": "Payment Orchestration environment to connect to.",
       "required": true,
       "value": "payments.sandbox.verygoodsecurity.app"
+    },
+    "ROUTE_ID": {
+      "description": "Route id from dashboard",
+      "required": true
+    },
+    "CUSTOMER_VAULT_ID_CHECKOUT_WEB_TESTS": {
+      "description": "vault ID for web",
```

Comment:
What does this description mean? We will be asking customers to fill in these fields so it's important that the descriptions are clear. We already ask for the Vault ID, so do we also need this?
![Uploading image.png…]()